synapseclient/client.py

```diff
@@ -118,7 +118,6 @@
     id_of,
     get_properties,
     MB,
-    memoize,
```

Comment:
Do you know if other pieces of code are using this function? If not, you can remove it all together.

Reply:
Thanks for the reminder Tom. I just checked again and it seems like this is the only place.